pyvista/plotting/widgets.py

```diff
@@ -85,7 +122,7 @@ def add_box_widget(
 
         rotation_enabled : bool, optional
             If ``False``, the box widget cannot be rotated and is
-            strictly orthogonal to the cartesian axes.
+            strictly orthogonal to the Cartesian axes.
```

Comment:
Good catch... this probably needs to be changed in 1000+ other places too

Reply:
Opening another PR to address this.